### ISSUE TITLE
[FEATURE] Enable logging by default

### ISSUE THREAD

**Issue by w13915984028** [harvester/harvester#2644]
**Is your feature request related to a problem? Please describe.**

Harvester will support logging feature.

**Describe the solution you'd like**

Harvester enables logging by default, after installation, it is available.

Due to existing limitation, `disable logging` is not available in v1.1.0.

**Describe alternatives you've considered**

**Additional context**

EPIC:
https://github.com/harvester/harvester/issues/577

HEP DOC:
https://github.com/joshmeranda/harvester/blob/logging/enhancements/20220525-system-logging.md

**Future enhancement**
[FEATURE] Harvester supports enable/disable managedchart in day2
https://github.com/harvester/harvester/issues/2719

With it, the `logging` can be enabled/disabled freely.

**Comment by w13915984028** [comment]
The resources usage of logging (https://github.com/harvester/harvester/issues/577). cc @joshmeranda @guangbochen @bk201

(tested in single-node cluster)
```
# kubectl top pod -A


NAMESPACE                   NAME                                                     CPU(cores)   MEMORY(bytes)   

cattle-fleet-local-system   fleet-agent-57497ff7dc-fmlct                             25m          552Mi           

cattle-fleet-system         fleet-controller-5746685958-dd7qx                        3m           166Mi           
cattle-fleet-system         gitjob-cc9948fd7-ng72b                                   1m           121Mi           

cattle-logging-system       rancher-logging-7b54b444c8-2djf4                         3m           118Mi           /// logging operator, one instance in cluster
cattle-logging-system       rancher-logging-root-fluentbit-q9zpz                     4m           30Mi            //// logging agent, per POD
cattle-logging-system       rancher-logging-root-fluentd-0                           5m           159Mi           //// logging server, one instance in cluster

cattle-monitoring-system    alertmanager-rancher-monitoring-alertmanager-0           1m           50Mi            // alertmanager

cattle-monitoring-system    prometheus-rancher-monitoring-prometheus-0               78m          530Mi           

cattle-monitoring-system    rancher-monitoring-grafana-d9c56d79b-jblt5               6m           162Mi
```

@joshmeranda when you have the resources usage of multi-node, please also post it, plus the grafana loki.

The `rancher-logging` creates a new namespace `cattle-logging-system`, which is also new to `Harvester`.

Enable `logging` by default will be good to business customers, the system has logging from beginning. It is also in same user experience as other Harvester features.
But it increases resource pressure to scenarios like edge deployment.  It will also be considered together with https://github.com/harvester/harvester/issues/2676 (`Add Harvester edge installation mode`) .  

Related topic in `Alertmanager`
https://github.com/harvester/harvester-installer/pull/322
https://github.com/harvester/harvester/issues/2518

**Comment by w13915984028** [comment]
FOR TEST:

In v1.1.0, test `logging/event/audit` output type of: Graylog, Splunk, they are required by customers;  
webhook server is for quick validation.

Set output manually :
How to send log to graylog:
https://github.com/w13915984028/harvester-develop-summary/blob/main/integrate-harvester-logging-with-log-servers.md

Test logging, audit, event with webhook server
https://github.com/w13915984028/harvester-develop-summary/blob/main/test-log-event-audit-with-webhook-server.md


Set output in UI:
https://github.com/harvester/harvester/issues/2646
[FEATURE] Config logging in Harvester Dashboard

https://github.com/harvester/harvester/issues/2749
[FEATURE] Harvester UI support config (cluster) flow & output for kube-audit log and event log

**Comment by lanfon72** [comment]
Closed as verified along with #2647, #2747, #2748
